packages/@react-aria/searchfield/src/useSearchField.ts

```diff
@@ -84,7 +85,7 @@ export function useSearchField(
     inputRef.current.focus();
   };
 
-  let {labelProps, inputProps} = useTextField({
+  let {labelProps, inputProps, isHovered} = useTextField({
```

Comment:
what should `isHovered` be spread on? we should put it into the appropriate return object to make it easy to use.
from looking down in the react-spectrum packages it belongs in the inputProps, however, it's a bit strange because it's not something we can spread on a dom node :-/
I wonder if it should be state that we pass into useTextField, that way we don't return it from the hook? that or maybe we should have a special object that is returned from our aria hooks that would contain things like isHovered.
Hold off on this suggestion until I can pull in some other opinions.

Reply:
See https://github.com/adobe/react-spectrum/pull/775#discussion_r456817604